pyproject.toml

```diff
@@ -18,8 +18,9 @@ authors = [
 keywords = []
 classifiers = [] # https://pypi.org/classifiers/
 dependencies = [
-  "grpcio == 1.51.3",
-  "grpcio_tools == 1.51.3"
+    "grpcio == 1.51.3",
+    "grpcio_tools == 1.51.3",
+    "pytest>=7.3.1",
```

Comment:
We should separate run-time deps from build-time ones. The official [PEP-518](https://peps.python.org/pep-0518/) doesn't allow to specify that (only build deps), but most tools running on pyproject.toml (Poetry, PDM, etc.), and most dependency managers in general (e.g.: npm), allow to manage build and dev dependencies separately.

Reply:
@epuertat 😀 I was moving uneasy while  adding `pytest` dependency to the runtime. I am not an expert in this area, let me know if  you have more a elegant recipe for that.